courses/dl2/imdb.ipynb

```diff
@@ -65,8 +108,17 @@
     "            labels.append(idx)\n",
     "    return np.array(texts),np.array(labels)\n",
     "\n",
-    "trn_texts,trn_labels = get_texts(PATH/'train')\n",
-    "val_texts,val_labels = get_texts(PATH/'test')"
+    "trn_texts,trn_labels = get_texts(PATH/'train') #read all the training reviews\n",
```

Comment:
since the function is called `get_texts` and the param is 'train', I think these comments aren't adding much. Let's only add comments where it's hard to know what's happening from the code, or why it's happening. (And normally markdown is best for that)